tests/unit/test_models/test_full_battery_models/test_lithium_ion/test_mpm_half_cell.py

```diff
@@ -20,12 +18,12 @@ def test_well_posed(self):
     def test_default_parameter_values(self):
         # check default parameters are added correctly
         model = pybamm.lithium_ion.MPM({"working electrode": "positive"})
-        self.assertEqual(
-            model.default_parameter_values["Positive minimum particle radius [m]"],
-            0.0,
+        assert (
+            model.default_parameter_values["Positive minimum particle radius [m]"]
+            == 0.0
         )
 
-    def test_lumped_thermal_model_1D(self):
+    def test_lumped_thermal_model_1_d(self):
```

Comment:
```suggestion
    def test_lumped_thermal_model_1D(self):
```

Let's keep the names of the functions the same

Reply:
How did these got changed 🤔 ? Must be something to do with pytestify maybe.